commons/model/src/main/java/io/github/microcks/util/Sanitizer.java

```diff
@@ -0,0 +1,23 @@
+package io.github.microcks.util;
```

Comment:
Can you add the standard copyright header and check your indentation?
I know, I know... we still have this topic on code formatter 😉

Reply:
@lbroudoux The indentation is 3 spaces right?

Reply:
We need to change the .editorconfig for the formatting

indent_size = 2 --> indent_size = 3

Reply:
Yes, I use the `.editorconfig` only for VS Studio when editing Typescript and HTML where indentation is 2 chars. We need to make things clear here ;-)